std_msgs/Header header
std_msgs/UInt32 robot_id
std_msgs/UInt32 pose_id
std_msgs/Float64[] data
